docs/whats-news/2022/2022-08.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 2022-08
+title: What's Newsletter August 2022
+slug: /2022-08
+---
+
+# What's New in Nimbus August 2022
+
+## Welcome
+
+The Nimbus team will have a **demo booth at #moz-aloha** All-Hands at the end of September! Come find us, chat about experimentation, and maybe even grab a sticker!
+
+## Exciting Changes
+
+### Mobile First Run
+
+Mobile First Run refers to the work done in Nimbus to allow experiments during an user's first run of the Firefox mobile app.
+- Instrument Nimbus SDK in telemetry in order to provide experimental treatments during the onboarding experience.
+  - This was affecting more than 100,000 clients and over 1.5M events/requests.
+
+### Improved Rollout Support
+
+A Rollout is an experiment where instead of configuring control and treatment branches, a feature is slowly rolled out to a population of clients. Experimenter now has improved support for Rollouts.
+- A client can:
+  - enroll in a rollout, receive feature values, send telemetry
+  - explicitly unenroll from the client UI
+  - be automatically unenrolled by unpublishing the record from RS
+  - enroll in a rollout for a feature, and then ALSO enroll in an experiment for the same feature, receive the experiment feature configuration while enrolled in the experiment, then be unenrolled from the experiment and fall back to the rollout feature configuration
+- User Interface
+  - Now just check a box to make your experiment a rollout (for supported clients).
```

Comment:
interesting, here are the list of supported clients 
```
    ROLLOUT_SUPPORT_VERSION = {
        Application.DESKTOP: Version.FIREFOX_105,
        Application.FENIX: Version.FIREFOX_105,
        Application.FOCUS_ANDROID: Version.FIREFOX_105,
        Application.IOS: Version.FIREFOX_105,
        Application.FOCUS_IOS: Version.FIREFOX_105,
    }
```
In short for every application we are supporting from 105 version

Reply:
Great! There was a note in one of the PRs about client support so I put that note in. I'll just remove that note, thanks!